python/private/py_console_script_binary.bzl

```
@@ -52,6 +52,7 @@ def py_console_script_binary(
         entry_points_txt = None,
         script = None,
         binary_rule = py_binary,
+        shebang = None,
```

```suggestion
        shebang = "",
```

Shouldn't this be a string and not `None`?